src/Behat/Config/Gherkin/Filter/FilterInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Behat\Config\Gherkin\Filter;
```

Comment:
The filters can also be applied on the suite level, so perhaps it would be better to remove the `Gherkin` part of the namespace? I know that the files that implement the filtering are in this Gherkin namespace but I don't think that forces us to use the same here

Reply:
done